install/compilation_options.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Compilation Options
+layout: default
+---
+<!--
+    Except where otherwise noted, content in this website is Copyright (c)
+    2015-2019, RTE (http://www.rte-france.com) and licensed under a
+    CC-BY-4.0 (https://creativecommons.org/licenses/by/4.0/)
+    license. All rights reserved.
+-->
+Dyna&omega;o compilation and launching can be customised through environment variables. `util/envDynawo.sh` script defines some environment variables than can be override by the user. The `myEnvDynawo.sh` script created at the first step of install is a simple way for the user to customise those **environment variables**. Some variables are  **mandatory** and some are **optional**, this page aims at providing information about all those variables.
+
+# Mandatory variables
+
+To launch `util/envDynawo.sh` script a list of variables needs to be defined.
+
+All the variables should be defined through an export `export VAR=VALUE`. For simplicity we recommend to define those in `myEnvDynawo.sh` like it is done in the install procedure of [Dyna&omega;o](/index.md).
+
+|Variable|Description|Value|
+|---------|-----------|-----------|
+|DYNAWO_HOME|Path of Dyna&omega;o source code.|PATH|
+|BUILD_TYPE|Compile Dyna&omega;o in Release or Debug.|Debug or Release|
+|CXX11_ENABLED|Should compiler use -std=c++11 or -std=c++98.|YES or NO|
+|OPENMODELICA_VERSION|Current OpenModelica version used by Dyna&omega;o.|1_9_4|
+|SRC_OPENMODELICA|Path of OpenModelica source code.|PATH|
+|INSTALL_OPENMODELICA|Path of OpenModelica install folder.|PATH|
+|USE_ADEPT|Should Dyna&omega;o use Adept as differentiation tool.|YES|
+
+# Optional variables
+
+Below is a list of variables that are optional to define in `myEnvDynawo.sh`. Usually we recommend to at least define:
+- NB_PROCESSORS_USED
+- RESULTS_SHOW
+- BROWSER
+
+|Variable|Description|Value|
+|---------|-----------|-----------|
+|NB_PROCESSORS_USED|Allow to use multiple cores for compilation.|1 or n|
+|BROWSER|Define your default browser command.|firefox or other|
+|RESULTS_SHOW|Should browser open at the end of simulation or tests coverage.|true or false|
+|DYNAWO_LOCALE|Enables to create a different local language for dictionaries.|en_GB|
```

Comment:
Perhaps indicate that at the moment, only en_GB is supported (I mean otherwise the developer needs to add its own dictionaries)